lib/src/index.js

```diff
@@ -0,0 +1,76 @@
+"use strict";
```

Comment:
Do we need to commit the js file? Can we have the circleci auto-publishing which builds/publishes the files, which guarantees that the js version matches the ts version, rather than potentially forgetting to update it. (also I may have got caught out and tried to review this file because it's not immediately obvious it's js rather than ts when starting at the bottom)